.travis.yml

```diff
@@ -6,52 +6,53 @@ git:
 
 #
 
+.env:
+  github_keys: &github_keys
+    - secure: "Zp76yFJIGFBRC4qFAGWNIyOHBSj8S/sKhAehLsYCTRPKWW4o30hg1jMhwp6XaWeyug9jRx+5CNevytr79p7XxRcKKWr1VQ4VWhg9mSSDeg0tNVZGNcu4z6kaDjfsEgHcYcNmgnJ4epo4Z8dZfJ4uCcD9mBrNcYGybiCAVFEok1LXXrUvgdUa/FgbPZGbMkG3EhR+8X7Z/8lLO/vkMpuYvpeTkrItck3PZxqjGIecTTvV/jfOB60SWRrlntTG0nDrtPCrHDHUy3rM43IFXMONzRlX9sqY8jv7yC4aImgLtnwg/Ii9hf+xoeL7+LpwIfG/+0GSBWnWR8pWSl7qgmRbsGZe32S/f0kIiAUaGaNpTzXyN07BP7mEIygLyCZETbBucafOlymb05ljsitXsYalogOuAHvkmkIeqTowTtqaVqnDUlSM9cRth31y0UIBeT3IIQr8kQhzFLVAKucWp4B1cP8gaC8jKsaI9dyPwN9ZvIoZsQCm7aWb7AGnVgg148byIXIK6QS9qen51YVTnRoPuJylABl6O6L6JV62o5aav8QUsV+UYKu/48wDkEI+y8oWfvEC+mdEZaOzUGm4lp4ucRCM/s+J3wtaPcyNG/SvOq4bX1+cgbP49UjxQkzvk2G9uzXbgwENEA9v4xCGewOt/PDyd/FTBTw/Qsalatpl8/o="
+
+#
+
 .node_stage: &node_stage
   language: node_js
   node_js: "10"
+  cache: yarn
   before_install:
     - curl -o- -L https://yarnpkg.com/install.sh | bash
     - export PATH="$HOME/.yarn/bin:$PATH"
   install:
     - yarn --frozen-lockfile
 
-.environments:
-  - &github_keys
-    - secure: "Zp76yFJIGFBRC4qFAGWNIyOHBSj8S/sKhAehLsYCTRPKWW4o30hg1jMhwp6XaWeyug9jRx+5CNevytr79p7XxRcKKWr1VQ4VWhg9mSSDeg0tNVZGNcu4z6kaDjfsEgHcYcNmgnJ4epo4Z8dZfJ4uCcD9mBrNcYGybiCAVFEok1LXXrUvgdUa/FgbPZGbMkG3EhR+8X7Z/8lLO/vkMpuYvpeTkrItck3PZxqjGIecTTvV/jfOB60SWRrlntTG0nDrtPCrHDHUy3rM43IFXMONzRlX9sqY8jv7yC4aImgLtnwg/Ii9hf+xoeL7+LpwIfG/+0GSBWnWR8pWSl7qgmRbsGZe32S/f0kIiAUaGaNpTzXyN07BP7mEIygLyCZETbBucafOlymb05ljsitXsYalogOuAHvkmkIeqTowTtqaVqnDUlSM9cRth31y0UIBeT3IIQr8kQhzFLVAKucWp4B1cP8gaC8jKsaI9dyPwN9ZvIoZsQCm7aWb7AGnVgg148byIXIK6QS9qen51YVTnRoPuJylABl6O6L6JV62o5aav8QUsV+UYKu/48wDkEI+y8oWfvEC+mdEZaOzUGm4lp4ucRCM/s+J3wtaPcyNG/SvOq4bX1+cgbP49UjxQkzvk2G9uzXbgwENEA9v4xCGewOt/PDyd/FTBTw/Qsalatpl8/o="
```

Comment:
@douglasduteil, it is this key!

Reply:
:thinking:

Reply:
Let merge this and release :)
I will look at the Travis build info to ensure that it's the right one :eye: